src/routes/Profile.tsx

```diff
@@ -119,6 +144,54 @@ function Profile(): JSX.Element {
               </div>
             </div>
           )}
+
+          {gameHistory != null && gameHistory.length > 0 ? (
+            <div className="mt-10 w-full flex items-start border-lightGrey flex-col sm:w-3/4">
+              <p className="text-2xl font-ubuntuBold  text-navy dark:text-aqua mb-3 ml-10">
+                Game history
+              </p>
+              <div className="card card-side bg-base-100 shadow-xl w-full flex flex-col p-2 sm:p-2 max-h-96 overflow-auto text-xs sm:text-base border border-lightGrey rounded dark:border-0">
+                <div className="flex w-full items-start justify-start text-center border-b-2 border-cloud dark:border-lightGrey">
+                  {["Opponent", "Result", "Score", "Date"].map(
+                    (header, index) => {
+                      return (
+                        <p
+                          className="w-1/4 font-ubuntuBold mb-2 text-center"
+                          key={index}
```

Comment:
while we have unique strings, it's better than `index`
```suggestion
                          key={header}
```